tests/testthat/test-Report_StudyInfo.R

```diff
@@ -1,4 +1,5 @@
 lStudy <- list(
+  # Are these variable names standardized?
   StudyID = "Unique Study ID",
```

Comment:
Specifically, are these terms in the `lStudy` object always fixed? Or can user put something like `studyid` ?

Reply:
The only fixed terms are defined in the [data_reporting workflow](https://github.com/Gilead-BioStats/gsm/blob/dev/inst/workflow/data_reporting.yaml), but I like the idea of applying a filter to additional terms that may or may not exist.

Reply:
These attribute names are Gilead-specific, and they match what comes out of `clindata::ctms_study`.